### PR TITLE
Fix: Add csrf verification to directory updates

### PR DESCRIPTION
**Pull request by acasajus**

### CODE REVIEW

File: templates/dashboard/directory.html [hunk 1]
```diff
@@ -173,18 +176,21 @@ <h2 class="h4 mb-1">New Directory</h2>
 
   <script>
     $(".delete-dir").on("click", function (e) {
-      let directory = $(this).parent().find(".dir-name").val();
+      console.log($(this));
```
Comment (cquintana92) [inline templates/dashboard/directory.html:179]: Should we remove this `console.log`?

Comment (acasajus) [reply templates/dashboard/directory.html:179]: Yes we should :+1:

Comment (acasajus) [inline templates/dashboard/directory.html:179]:
```suggestion
```

Comment (acasajus) [reply templates/dashboard/directory.html]: Done

**Review by cquintana92 — APPROVED** [review]

File: app/dashboard/views/directory.py [hunk 2]
```diff
@@ -33,6 +56,12 @@ def directory():
...
     new_dir_form = NewDirForm()
+    toggle_dir_form = ToggleDirForm()
+    update_dir_form = UpdateDirForm()
+    update_dir_form.mailbox_ids.choices = [
+        (str(mailbox.id), str(mailbox.id)) for mailbox in mailboxes
+    ]
+    delete_dir_form = DeleteDirForm()
 
     if request.method == "POST":
         if request.form.get("form-name") == "delete":
```
Comment (nguyenkims) [inline app/dashboard/views/directory.py:67]: should we use `delete_dir_form` here to check the csrf?

**Pull request merged**